specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json

```diff
@@ -897,6 +897,14 @@
           },
           "description": "A collection of references to subnets."
         },
+        "flowLogs": {
```

Comment:
Do you need  to update the example of this new property?

Reply:
No, I don't think so. This property becomes populated only if there is flowLog resource created for this NSG. Plus, usually, linked properties are not specified in examples. IN NSG, besides flowLog there are a lot of linked properties (like networkInterfaces, f.e.), but none of them is in examples.

Reply:
I see. Thanks